planning/behavior_velocity_planner/src/scene_module/crosswalk/scene_crosswalk.cpp

```diff
@@ -468,8 +468,11 @@ boost::optional<std::pair<size_t, PathPointWithLaneId>> CrosswalkModule::findNea
     const auto & obj_vel = object.kinematics.initial_twist_with_covariance.twist.linear;
     const auto obj_uuid = toHexString(object.object_id);
 
-    if (!isTargetType(object)) {
+    if (isVehicle(object)) {
```

Comment:
https://github.com/autowarefoundation/autoware.universe/blob/5c19ee513deec69ae8cbc6b340f66e5c9dce5686/planning/behavior_velocity_planner/src/scene_module/crosswalk/scene_crosswalk.cpp#L929-L932
The type of object is already checked here.

Reply:
I think it's better to use isVehicle() function instead of L929-L932

Reply:
You are right :+1: I'll fix it.

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/1319/commits/73963315046094b067de0f3fedca37941e9d5999